packages/uni_app/lib/view/bug_report/widgets/form.dart

```diff
@@ -245,7 +245,10 @@ class BugReportFormState extends State<BugReportForm> {
     setState(() {
       _isButtonTapped = true;
     });
-    final faculties = PreferencesController.getUserFaculties();
+
+    final session = Provider.of<SessionProvider>(context, listen: false).state;
```

Comment:
Why not access session through `PreferencesController`?

Reply:
Because the PreferencesController only has the session set when the login is done. The cookies are not guaranteed to be valid and you should always use an AuthenticationController with that session.